### PR TITLE
Test multi cloud configurations

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

File: roles/test_multi_cloud/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            collectors:
+            - collectorType: collectd
+              subscriptionAddress: collectd/cloud1-telemetry
+            - collectorType: ceilometer
+              subscriptionAddress: anycast/ceilometer/cloud1-metering.sample
+        - name: cloud2
+          events:
+            collectors:
+            - collectorType: collectd
+              subscriptionAddress: collectd/cloud1-notify
```
Comment (csibbitt) [inline roles/test_multi_cloud/tasks/main.yml:30]:
```suggestion
              subscriptionAddress: collectd/cloud2-notify
```

File: roles/test_multi_cloud/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            - collectorType: ceilometer
+              subscriptionAddress: anycast/ceilometer/cloud1-event.sample
```
Comment (csibbitt) [inline roles/test_multi_cloud/tasks/main.yml:32]:
```suggestion
              subscriptionAddress: anycast/ceilometer/cloud2-event.sample
```

File: roles/test_multi_cloud/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          metrics:
+            collectors:
+            - collectorType: collectd
+              subscriptionAddress: collectd/cloud1-telemetry
```
Comment (csibbitt) [inline roles/test_multi_cloud/tasks/main.yml:36]:
```suggestion
              subscriptionAddress: collectd/cloud2-telemetry
```

File: roles/test_multi_cloud/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            - collectorType: ceilometer
+              subscriptionAddress: anycast/ceilometer/cloud1-metering.sample
```
Comment (csibbitt) [inline roles/test_multi_cloud/tasks/main.yml:38]:
```suggestion
              subscriptionAddress: anycast/ceilometer/cloud2-metering.sample
```

File: roles/test_multi_cloud/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+          oc exec -n service-telemetry prometheus-default-0 -- wget -qO- 'localhost:9090/api/v1/query?query=collectd_uptime{service="default-cloud1-coll-meter"}'
+      register: prometheus_output_cloud1
+      changed_when: false
+
+    - name: Check if the output contains success
+      ansible.builtin.debug:
+        msg: 
+          - "Query for cloud1 was successful."
+          - "{{ prometheus_output_cloud1.stdout }}"
+      when: '"success" in prometheus_output_cloud1.stdout'
```
Comment (csibbitt) [inline roles/test_multi_cloud/tasks/main.yml:86]: This does not prove that any metrics were processed. Here is the output from a brand new STF install with no OSP attached and no metrics in the DB.

```
$ oc exec -n service-telemetry prometheus-default-0 -- wget -qO- 'localhost:9090/api/v1/query?query=collectd_uptime{service="default-cloud1-coll-meter"}'
{"status":"success","data":{"resultType":"vector","result":[]}}
```

I get the same output with `service="ThisCloudDoesntExist"`

```
$ oc exec -n service-telemetry prometheus-default-0 -- wget -qO- 'localhost:9090/api/v1/query?query=collectd_uptime{service="ThisCloudDoesntExist"}'
{"status":"success","data":{"resultType":"vector","result":[]}}
```

**Comment by mgirgisf** [comment]
This functional Test will not be used until we add multi-cloud to the integration Job.

**Pull request closed without merging**